canonical-web-and-design/snapcraft.io
# Included install instructions specific for the distro here

Fixes #1996 

## QA Steps

1. Pull the branch or visit the demo URL
2. Visit any snap you want
3. Scroll down the bottom to see links to the relevant distributions you want to get snaps support
4. Visit the link
5. You should see instructions on how to install snap support on the distribution
6. #winning
